code/modules/client/preferences_savefile.dm

```diff
@@ -73,6 +73,19 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		if(clientfps == 0)
 			clientfps = -1
 
+	if (current_version < 38)
+		var/found_block_movement = FALSE
+
+		look_through_bindings:
+			for (var/list/key in key_bindings)
+				for (var/bind in key)
+					if (bind == "block_movement")
+						found_block_movement = TRUE
+						break look_through_bindings
+
+		if (!found_block_movement)
+			LAZYADD(key_bindings["Ctrl"], "block_movement")
```

Comment:
```suggestion
			LAZYADD(key_bindings["Alt"], "block_movement")
```
Default is now Alt

Reply:
This was intentional. *The default* is. What people have been used to for years isn't--this was intended to make players not feel like there was a regression.

Reply:
Ah, that's fair.